tests/mocks/Identity/index.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  identifier: 'H2pb35GtKpjLinncBYeMsXkdDYXCbsFzzVmssce6pSJ1',
+  identifierBytes: [238, 50, 96, 211, 63, 140, 169, 225, 76, 97, 8, 6, 212, 109, 184, 253, 51, 67, 82, 42, 208, 228, 72, 192, 215, 137, 161, 88, 144, 55, 244, 230],
+  identifierSetted: 'H3pb35GtKpjLinncBYeMsXkdDYXCbsFzzVmssce6pSJ1',
+  balance: 100,
```

Comment:
I think it would be better to separate mock data from expects

It could be just an assert in the test case